specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json

```diff
@@ -7623,6 +7681,112 @@
           }
         }
       }
+    },
+    "DataWarehouseQueries": {
+      "description": "The response to a list datawarehouse queries.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DataWareHouseQueriesProperties",
+          "description": "Resource properties."
+        }
+      }
+    },
+    "DataWareHouseQueriesProperties": {
+      "description": "Properties of a datawarehousequeries.",
+      "type": "object",
+      "properties": {
+        "queryId": {
+          "description": "Requested QueryId. of datawarehousequeries",
+          "type": "string",
+          "readOnly": true
+        },
+        "startTime": {
+          "description": "The start time for queries datawarehousequeries",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "endTime": {
+          "description": "The end time for queries",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "duration": {
+          "description": "The duration of requested queries.",
+          "type": "number",
+          "format": "int32",
+          "readOnly": true
+        },
+        "status": {
+          "description": " Status of dataWarehouseQueries.",
+          "type": "string",
+          "readOnly": true
+        },
+        "queryText": {
+          "description": " QueryText Message",
+          "type": "string",
+          "readOnly": true
+        },
+        "importance": {
+          "description": "dataWarehouseQueries Importance",
+          "type": "string",
+          "readOnly": true
+        },
+        "groupName": {
+          "description": "Requested GroupName ",
+          "type": "string",
+          "readOnly": true
+        },
+        "classifierName": {
+          "description": "Requested ClassifierName",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceAllocationPercentage": {
+          "description": "AllocationPercentage of the Resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "error": {
+          "description": "The operation error message.",
+          "type": "string",
+          "readOnly": true
+        },
+        "login": {
+          "description": "Login Details",
+          "type": "string",
+          "readOnly": true
+        },
+        "progress": {
+          "description": "Progress value",
+          "type": "number",
+          "format": "int32",
+          "readOnly": true
+        },
+        "sessionId": {
+          "description": "SessionId for Resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceClass": {
```

Comment:
What are the potential values of resource class?

Reply:
@bansalaseem added values for resource class